Makefile

```diff
@@ -1,39 +1,24 @@
-TGIS_IMAGE=quay.io/opendatahub/text-generation-inference:stable-bafd218
-
-.PHONY: default
+.PHONY: default refresh-poetry-lock-files
 
 default:
-	podman build -t caikit-tgis-serving:$$(git rev-parse --short HEAD) .
-
+	podman build \
```

Comment:
Does it make sense to allow th users to use something other than podman ?

```suggestion
# Container Engine to be used for building images
ENGINE ?= podman

default:
	$(ENGINE) build \
```

Reply:
Wondering the same, but for now I'd leave this as is, possibly adding this feature in another PR.

Reply:
Yep, future PR